pino.js

```diff
@@ -258,7 +259,7 @@ Pino.prototype.child = function child (bindings) {
   return new Pino(
     bindings.level || this.level,
     this.stream,
-    this.serializers,
+    extend(this.serializers, bindings.serializers),
```

Comment:
My goal is, there should be no perf degradation _if `bindings.serializers` is not passed in_. If there are by setting `serializers`, it's fine as it is an edge case anyway.